drivers: comparator: add comparator support on Renesas RA LVD

thenguyenyf opened the pull request:
Initial support for Renesas RA Comparator driver running on LVD HWIP

Commit:
drivers: comparator: Add comparator LVD driver support for RA

Add comparator support for RA with LVD module

Signed-off-by: Minh Tang <minh.tang.ue@bp.renesas.com>

Commit:
dts: renesas: ra: Add dts node for LVD support

Add dts node for LVD support on RA8 MCUs

Signed-off-by: Minh Tang <minh.tang.ue@bp.renesas.com>

Commit:
soc: renesas: ra: Support NMI Handler for RA8P1 SoC

Add register NMI_Handler for NMI when CONFIG_RUNTIME_NMI is enabled on
RA8P1

Signed-off-by: Minh Tang <minh.tang.ue@bp.renesas.com>

thenguyenyf committed:
manifest: update rev of hal_renesas to latest

Update hal_renesas revision to add r_lvd support

Signed-off-by: The Nguyen <the.nguyen.yf@renesas.com>

bjarki-andreasen reviewed (commented):
Code looks good, could you extend the comparator build all and comparator api test suites to test the driver?

thenguyenyf committed:
tests: drivers: build_all: add renesas_lvd test configurations

Add build test configuration for renesas_lvd on Renesas RA boards
that currently support the comparator driver on LVD.

Signed-off-by: The Nguyen <the.nguyen.yf@renesas.com>

bjarki-andreasen reviewed (dismissed):
Looks good

thenguyenyf commented:
Last push just to rebase and update west manifest revision due to it was merged
